.gitignore

```diff
@@ -0,0 +1 @@
+zookeeper-docker.iml
```

Comment:
Please keep this in your local gitignore because it's specific to your environment.

Reply:
Good catch! Pull Request updated.
Thanks for letting me know.